Framework/API/inc/MantidAPI/ADSValidator.h

```diff
@@ -0,0 +1,79 @@
+#ifndef MANTID_API_ADSVALIDATOR_H_
+#define MANTID_API_ADSVALIDATOR_H_
+
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "MantidKernel/TypedValidator.h"
+#include "MantidAPI/DllConfig.h"
+
+namespace Mantid {
+namespace API {
+
+using namespace Mantid::Kernel;
```

Comment:
`using` declarations should not be anywhere in the header. Please can we remove this before it gets to master.